app/views/layouts/_flashes.html.erb

```diff
@@ -0,0 +1,7 @@
+<% unless flash.empty? %>
+  <div class="flashes">
+    <% flash.each do |type, message| %>
+      <div class="<%= flash_htmlclass_for type %>"><%= message.to_s %></div>
```

Comment:
sorry to ask this so late in the game but
1. what happens (in terms of the css class) when anything other than type :success or :error is called. 
  - Do we have just an empty class attribute?
  - Is this valid html?
  - Do we want to have a default we return such as class="flash", and if we do do we want to add this class to the other cases too.